static/app/views/performance/content.tsx

```diff
@@ -250,6 +252,19 @@ class PerformanceContent extends Component<Props, State> {
     );
   }
 
+  renderLandingV3() {
+    return (
+      <PerformanceLanding
+        eventView={this.state.eventView}
+        setError={this.setError}
+        handleSearch={this.handleSearch}
+        handleTrendsClick={() => this.handleTrendsClick()}
```

Comment:
Should `handleTrendsClick` be an arrow function so you don't have to wrap it here? I believe wrapping it here would result in excessive re-renders (though it might not be all that noticeable)